docs/src/content/docs/ops/guides/starting-with-aws.adoc

```diff
@@ -304,28 +304,29 @@ For more information on the available configuration options and fetching the cha
 
 === Accessing the XTDB Cluster
 
-NOTE: As it will take some time for the XTDB nodes to be marked as ready (as they need to pass their initial startup checks), and AWS will need to provision a Load Balancer for the service, it may take a few minutes for the XTDB cluster to be accessible.
+NOTE: As it will take some time for the XTDB nodes to be marked as ready (as they need to pass their initial startup checks) it may take a few minutes for the XTDB cluster to be accessible.
 
-Once the XTDB cluster is up and running, you can access it via the LoadBalancer service that was created.
+NOTE: The xtdb service is only available via ClusterIP be default so as to not expose the service publicly
 
-To get the external IP of the LoadBalancer service, run:
+Once the XTDB cluster is up and running, you can access it via the ClusterIP service that was created.
+
+To port forward the service locally
 ```bash
-kubectl get svc xtdb-service --namespace xtdb-deployment
+kubectl port-forward service/xtdb-service --namespace xtdb-deployment 8080:8080
```

Comment:
Could you update the instructions within the various NOTES.txt files too? See: https://github.com/xtdb/xtdb/blob/main/aws/helm/templates/NOTES.txt